login-workflow/docs/ContactSupport.md

```diff
@@ -0,0 +1,71 @@
+| divider                               | Optional divider which appears above `Actions`            | `boolean`                                                         | no       | `true`                                                                      |
+| ContactSupportContent                 | Element to display the contact support content            | `JSX.Element`                                                     | no       | `<CardContent>...</CardContent>`                                            |
+| CardHeaderProps                       | Props to pass to `CardHeader`                             | `CardHeaderProps`                                                 | no       |                                                                             |
+| CardContentProps                      | Props to pass to `CardContent`                            | `CardContentProps`                                                | no       |                                                                             |
+| CardActionsProps                      | Props to pass to `CardActions`                            | `CardActionsProps`                                                | no       |                                                                             |
+| phoneNumber                           | The text to override contact phone number                 | `string`                                                          | no       | 1-800-123-4567                                                              |
+| emailId                               | The text to override contact email id                     | `string`                                                          | no       | something@email.com                                                         |
+| contactSupportQuestion                | The text to override contact support question             | `string`                                                          | no       | General Questions                                                           |
+| contactSupportMessage                 | The text to override contact support message              | `string`                                                          | no       | For questions, feedback, or support please email us at something@email.com. |
+| contactSupportEmergency               | The text to override contact support emergency support    | `string`                                                          | no       | Emergency Support                                                           |
+| contactSupportTechnicalAssistance     | The text to override contact support technical assistance | `string`                                                          | no       | For technical support, please call 1-800-123-4567.                          |
```

Comment:
Should all the `string` defaults be shown as strings with quotes? maybe double quotes for human readability? e.g.
```suggestion
| phoneNumber                           | The text to override contact phone number                 | `string`                                                          | no       | "1-800-123-4567"                                                              |
| emailId                               | The text to override contact email id                     | `string`                                                          | no       | "something@email.com"                                                         |
| contactSupportQuestion                | The text to override contact support question             | `string`                                                          | no       | "General Questions"                                                           |
| contactSupportMessage                 | The text to override contact support message              | `string`                                                          | no       | "For questions, feedback, or support please email us at something@email.com." |
| contactSupportEmergency               | The text to override contact support emergency support    | `string`                                                          | no       | "Emergency Support"                                                          |
| contactSupportTechnicalAssistance     | The text to override contact support technical assistance | `string`                                                          | no       | "For technical support, please call 1-800-123-4567."                         |
```